src/pydicom/pixels/decoders/base.py

```diff
@@ -1386,19 +1415,26 @@ def _as_buffer_encapsulated(
         Returns
         -------
         bytes | bytearray
-            A buffer-like containing the decoded pixel data.
+            A buffer-like containing the decoded pixel data. For single bit
+            images (Bits Allocated of 1), pixels are always returned
```

Comment:
*Bits Allocated* please

Reply:
Addressed in [e9ea3de](https://github.com/pydicom/pydicom/pull/2216/commits/e9ea3deb538f6786f5a80e0e826d7aaf7a40678c)